tdrs-backend/tdpservice/data_files/views.py

```diff
@@ -97,7 +97,13 @@ def filter_queryset(self, queryset):
         """Only apply filters to the list action."""
         if self.action != 'list':
             self.filterset_class = None
-        return super().filter_queryset(queryset)
+
+        if self.request.query_params.get('file_type') == 'ssp-moe':
```

Comment:
Doing specific filtering should be done in get_queryset instead of filter_queryset.